test/cppsuite/configs/bounded_cursor_perf_default.txt

```diff
@@ -0,0 +1,20 @@
+# Configuration for bounded_cursor_perf.
+# The test will repeatedly call next() and prev() operations and track performance of the operations
+# done with both range cursor and regular cursors. 
+# This config will have a 1 minute duration, with only one read thread. 
+
+duration_seconds=10,
+cache_size_mb=1000,
+workload_generator=
+(
+    populate_config=
```

Comment:
Can we add an extra config here for testing the perf plotter? Effectively we can validate using the default test if we produce validate json output from the two components combined.
```
runtime_monitor=
(
    cache_hs_insert=
    (
        max=10,
        max=100000000,
        postrun=true,
        save=true,
    ),
    cc_pages_removed=
    (
        max=10000000,
        postrun=true,
        save=true,
    ),
)
```

Reply:
sounds good!